### PR TITLE
Bump kover to 0.9.7

### PR DESCRIPTION
**Pull request by KCeh**
## Summary

Simple change, bumping kover and migrating config to new version

**Related issue**: 

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

## Checklist

- [ ] I have performed a self-review of my own code.
- [ ] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [ ] I have updated the documentation to reflect my changes (if applicable).

### CODE REVIEW

File: dbinspector/build.gradle [hunk 1]
```diff
@@ -142,10 +113,39 @@ protobuf {
...
+                    "com.infinum.dbinspector.ui.shared.edgefactories.*",
+                    "com.infinum.dbinspector.ui.shared.listeners.*",
+                    "com.infinum.dbinspector.ui.shared.searchable.*",
+                    "com.infinum.dbinspector.ui.shared.views.*",
+                    "com.infinum.dbinspector.extensions.*",
+                    "com.infinum.dbinspector.databinding.*"
+                )
+            }
+        }
+    }
```
[the remark this answers is not in the document]
Comment (KCeh) [reply dbinspector/build.gradle:148]: I will recheck
